dev/io.openliberty.jakartaee10.internal_fat/fat/src/io/openliberty/jakartaee10/internal/tests/EE10Features.java

```diff
@@ -282,6 +282,8 @@ private static Set<String> getExtendedCompatibleFeatures(Set<String> compatibleF
         // remove logAnalysis-1.0.  It depends on hpel being configured
         features.remove("logAnalysis-1.0");
 
+        feature.remove("audit-2.0");
```

Comment:
```suggestion
        features.remove("audit-2.0");
```
This is what caused your build break.